playground.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Playground</title>
+    <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+    <div class="avatar avatar-lg-size">
+        <img src="Stylesheet/Images/avatar.jpg" class="img-responsive img-round" alt="avatar">
```

Comment:
Move your images to separate folder. Dont include them in stylesheets folder

Reply:
Okay will change that